src/libraries/Microsoft.PowerFx.Core/IR/IRNodeVisitor.cs

```diff
@@ -7,36 +7,36 @@ namespace Microsoft.PowerFx.Core.IR
 {
     internal abstract class IRNodeVisitor<TResult, TContext>
     {
-        public abstract TResult Visit(TextLiteralNode node, TContext context);
+        public abstract TResult Visit(TextLiteralNode node, TContext context, StackMarker stackMarker);
```

Comment:
This is the wrong approach. We should not be modifying this interface in this way.

Reply:
Changing this over to the proper way to implement the interface.